### ISSUE TITLE
Inputs are not selectable

### ISSUE THREAD

**Issue by zhigang1992** [dexterleng/vimac#270]
<img width="622" alt="image" src="https://user-images.githubusercontent.com/1057756/99214692-d3513d00-280b-11eb-923a-f41f233c547b.png">

Seems on multiple sites in Safari that inputs are not able to be selected.

Is there a way around that?

**Comment by blackketter** [comment]
Interesting.  I'm seeing this in Safari (in Big Sur):

<img width="593" alt="Screen Shot 2020-11-15 at 9 19 54 PM" src="https://user-images.githubusercontent.com/131679/99215665-78c2cd00-2788-11eb-8149-054456fb6094.png">

**Comment by zhigang1992** [comment]
Yeah, I'm on Safari Big Sur as well.
I've just tested with all the extensions disabled, it's still the same.

**Comment by zhigang1992** [comment]
I was just showing for me on twitter, I got excited and checked GitHub again. And it still not showing, then I go back to twitter. It stopped showing as well.

**Comment by zhigang1992** [comment]
<img width="622" alt="image" src="https://user-images.githubusercontent.com/1057756/99497141-73e06200-29b0-11eb-8fc0-0ea125ea0c07.png">

it works after turn on this option, this options was off by default. 

@dexterleng

**Comment by zhigang1992** [comment]
@dexterleng this is happening again, and with latest version, we no longer have access to these options?

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/1057756/102964056-a6d4c300-4525-11eb-9daf-7d9c5c79f516.png">

**Comment by dexterleng** [comment]
The new release will address this issue. Refer to #280 for how it is fixed. Do reopen this PR if you continue facing this issue.

**Comment by zhigang1992** [comment]
It's working perfectly!

Thank you for your hard work. 🙇🏻
